deltachat-ios/Controller/ConnectivityViewController.swift

```diff
@@ -78,20 +78,19 @@ class ConnectivityViewController: WebViewViewController {
                 .appending(String.localized("no_data"))
         }
 
-        if timestamps.isEmpty || timestamps.count == 1 {
-            // FIXME: for timestamp == 1, that is just okay if the timestamp is not too old
+        var averageDelta: Double = 0
+        if timestamps.isEmpty {
+            // this should not happen:
+            // the array should not be empty as old notifications are only removed if a new one is added
             return "<span class=\"red dot\"></span>"
                 .appending(title)
-                .appending(String.localized("notifications_not_working"))
+                .appending("Bad Data")
```

Comment:
shall we translate that?

Reply:
it is an error message for a case that should not appear at all, i do not think, it make much sense to have that translated.